internal/application/create-code-revew.go

```diff
@@ -12,9 +12,13 @@ type Context struct {
 }
 
 func (a *App) CreateCodeReview() {
-	comment := a.aiModel.GetComment()
-	err := a.codeRepositoryProvider.CreateComment(comment)
+	changes, err := a.codeRepositoryProvider.GetPullRequestChanges()
 	if err != nil {
 		log.Fatalf("Error: %v\n", err)
 	}
+	err = a.codeRepositoryProvider.CreateComment(changes)
+	if err != nil {
+		log.Fatalf("Error: %v\n", err)
+	}
+
 }
```

Comment:
Code Review 2024-08-18 02:28:50 - New lines 2 to 6:

	err = a.codeRepositoryProvider.CreateComment(changes)
	if err != nil {
		log.Fatalf("Error: %v\n", err)
	}